src/www/ui/api/Controllers/UploadController.php

```diff
@@ -417,11 +417,11 @@ private function changeUpload($request, $response, $args, $isCopy)
    */
   public function postUpload($request, $response, $args)
   {
-    $uploadType = $request->getHeaderLine('uploadType');
+    $reqBody = $this->getParsedBody($request);
+    $uploadType = $reqBody['uploadType'] ?? null;
     if (empty($uploadType)) {
       $uploadType = 'vcs';
```

Comment:
As per the OpenAPI doc, `uploadType` is a required field and thus, should throw appropriate exception if empty.